server/index.js

```diff
@@ -12,6 +12,24 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 
+app.get('/api', (req, res) => {
+  // Stocks.aggregate([{ $sample: {size: 9}}]).then(function (res) {
+  //   console.log(res);
+  // })
```

Comment:
Take these out, store them in your journal with a little descriptor if you really need them